content/articles/ordering-comodo-certificate.markdown

```diff
@@ -14,48 +14,53 @@ categories:
 
 ---
 
-DNSimple provides an SSL certificate interface you can use to purchase a new SSL certificate.
+DNSimple provides an SSL certificate interface you can use to order a new SSL certificate.
 
-All DNSimple SSL certificates are valid for one year from their issue date. Sixty days before the certificate expires you will begin receiving renewal notices.
+All traditional SSL certificates sold by DNSimple are valid for at least one year from their issue date. Sixty days before the certificate expires you will begin receiving renewal notices.
 
 
-## Purchase requirements
+## Before you start
 
-To purchase an SSL certificate you need a DNSimple account. In order to finalize the purchase, our system requires the account to be subscribed to a DNSimple plan, however you can unsubscribe later on once the certificate is successfully purchased and issued.
+To order an SSL certificate you need a DNSimple account. In order to finalize the order, the account [must be subscribed](/articles/account-activation) to a DNSimple plan.
 
-**You are not required to transfer or host a domain with us to purchase an SSL certificate**. You can purchase an SSL certificate for a domain, even if the domain is hosted elsewhere or registered with another registrar. However, an [active account](/articles/account-activation) is required in order to buy it.
+<note>
+You are not required to keep an active subscription to access the certificate, however some features such as certificate renewal or auto-renewal will not be available when you cancel your subscription.
+</note>
 
-Please be sure you comply with the all the [technical requirements](/articles/getting-started-ssl-certificates/#requirements) in our [Getting Started with SSL Certificates](/articles/getting-started-ssl-certificates) page before purchasing the certificate.
+You can order an SSL certificate for a domain, even if the domain is hosted elsewhere or registered with another registrar. **You are not required to transfer or host a domain with us to purchase an SSL certificate**.
```

Comment:
Perhaps change to "purchase a Comodo SSL certificate". I know it's the Comodo page, but it might not hurt to be specific in case people remember reading this and think it applies to all SSLs.